consult-dir.el

```diff
@@ -78,11 +78,15 @@
 
 (declare-function projectile-load-known-projects "projectile")
 (declare-function project--read-project-list "project")
+(declare-function tramp-parse-sconfig "tramp")
+(declare-function docker-tramp--parse-running-containers "docker-tramp")
```

Comment:
I would like to avoid adding built-in support for any more packages that are not part of Emacs (projectile being an early exception). We can include the `docker-tramp` source as an example in the README.